### PR TITLE
Step number 2 is here

### PR DESCRIPTION
**Pull request by tudor1703**

### CODE REVIEW

**Comment by tudor1703** [comment]
The second commit is from step 3, I forgot to change branches.

**Review by Georg1703 — CHANGES_REQUESTED** [review]
Will be nice to see file env_example where will be names of variables that i should setup in .env so the project will start successfully.

db should never get to github, sqlite is also a mini database so exclude it from git.

File: mysite/mysite/settings.py [hunk 1]
```diff
@@ -85,7 +85,7 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'NAME': os.path.join(BASE_DIR, os.getenv('DATABASE_PATH')),
```
Comment (Georg1703) [inline mysite/mysite/settings.py:88]: De ce ai schimbat asta aici dar in .env example nu ai inclus DATABASE_PATH ?

Comment (Georg1703) [reply mysite/mysite/settings.py:88]: Eu as zice ca nu e nevoie de asta aici si nici de datele legate de database in env example, oricum sqlite nu e o baza de date adevarata, o sa facem asta pe urma.

**Review by Georg1703 — CHANGES_REQUESTED** [review]

File: mysite/mysite/settings.py [hunk 2]
```diff
@@ -85,7 +85,7 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, os.getenv('DATABASE_PATH')),
+        'NAME': BASE_DIR / 'db.sqlite3',
```
Comment (Georg1703) [inline mysite/mysite/settings.py]: De ce ai schimbat asta ? Doar ai adaugat DATABASE_PATH in env.

**Review by Georg1703 — CHANGES_REQUESTED** [review]

**Pull request merged**